src/solvers/qbf/qbf_bdd_core.cpp

```diff
@@ -190,7 +203,7 @@ propt::resultt qbf_bdd_coret::prop_solve()
 {
   {
     std::string msg=
-      solver_text() + ": "+
+      solver_text()+": "+
       std::to_string(no_variables())+" variables, "+
       std::to_string(matrix->nodeCount())+" nodes";
     messaget::status() << msg << messaget::eom;
```

Comment:
Nit pick: would you mind changing this code to repeatedly use `<<` instead of first building a string and then outputting it in one piece?